pkg/cliplugins/bind/plugin/bind_compute.go

```diff
@@ -248,7 +249,8 @@ func (o *BindComputeOptions) applyAPIBinding(ctx context.Context, client kcpclie
 		clusterName, name := logicalcluster.New(export).Split()
 		apiBinding := &apisv1alpha1.APIBinding{
 			ObjectMeta: metav1.ObjectMeta{
-				Name: apiBindingName(clusterName, name),
+				Name:   apiBindingName(clusterName, name),
+				Labels: map[string]string{apisv1alpha1.LabelAPIBindingKindKey: workloadv1alpha1.LabelKindAPIBindingValue},
```

Comment:
I assume that we should now take in account the case when the `APIBinding` exists but does **not** have the `workload` kind (because having been bound previously with the regular `kcp bind apiexport` command). In such a case we should patch it with the new label.